File: templates/python/hvdc_facts_stage_step.py
"""hvdc_facts_stage_step: HVDC 与 FACTS阶段阶梯曲线（step 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='HVDC and FACTS analysis: stage step curve'):
    return make_template_figure('step_curve', seed=3717, title=title, domain='HVDC and FACTS analysis', topic='stage step curve')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/hvdc_facts_stage_step.m
function fig = hvdc_facts_stage_step()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('step_curve', 3717, 'HVDC and FACTS analysis: stage step curve', 'HVDC and FACTS analysis', 'stage step curve');
end
